### ISSUE TITLE
More buffers than expected

### ISSUE THREAD

**Issue by valery-reznic** [wsdjeg/vim-fetch#14]
I am using vim-fetch like following:
```shell
vim `grep pattern -Irn . | awk '{print $1}'
```

So vim invoked like
```shell
vim file1:12 file1:23 file2:33 ...
```
Turns out following:
1. When vim started it open file1 at line 12
2. After first `:n` it  open file1 at line:23 (as expected)
3. After second `:n` instead of opening file2 at line 33 it again opened file1 at line:23
4. Third `:n` open file2 at line 33 (expected after second :n)

**Comment by kopischke** [comment]
Huh, that is odd. Could you please deactivate _vim-fetch_ and repeat the experience? You will get a bunch of spurious empty buffers called `file1:12` etc. I am interested in learning if the buffers are duplicated as above without _fetch_ resolving them too.

**Comment by valery-reznic** [comment]
I deactivated vim-fetch and run
vim file1:12 file1:23

I got empty file (file1:12). 
After :n I got empty file (file1:23)
After second :n I got "E165: Cannot go beyond last file"
Everything as expected

**Comment by valery-reznic** [comment]
Something new here? Do you need more info from me?

**Comment by kopischke** [comment]
Yes and no, sorry for the delay: I meant to test why you consider `E165` “normal” by checking if (and  how) it also happens on my system, but I have precious little time for Vimming ATM.

**Comment by kopischke** [comment]
Possible duplicate of #17.

**Comment by kopischke** [comment]
Fixed in ef8a8e78f9bb724ca0fd2bd0b4e39d3e12487832.
